src/core_processor/fps/fps_counter.py

```diff
@@ -0,0 +1,60 @@
+import time
+from typing import Dict, List
+
+
+class FPSCounter:
```

Comment:
This class encapsulates the logic to count FPS. We can then use this class in code that does processing without recreating the same FPS Counting logic.